src/rl_coach_2020_v2/hyperparams.json

```diff
@@ -0,0 +1,17 @@
+{
+	"batch_size": 64,
+	"beta_entropy": 0.01,
+	"discount_factor": 0.999,
+	"e_greedy_value": 0.05,
+	"epsilon_steps": 10000,
+	"exploration_type": "categorical",
+	"loss_type": "mean squared error",
+	"lr": 0.0003,
+	"num_episodes_between_training": 20,
+	"num_epochs": 10,
+	"stack_size": 1,
+	"term_cond_avg_score": 100000.0,
+	"term_cond_max_episodes": 10000,
+	"pretrained": 0
```

Comment:
Can this be true/false please to be more user friendly?

Reply:
Done, it is case insensitive.